packages/client/components/InsightsCharts.tsx

```diff
@@ -194,6 +200,9 @@ const makeOptions = (title: string, subtitle: string) => {
         grid: {
           display: false,
           drawBorder: false
+        },
+        ticks: {
+          precision: 0
```

Comment:
+2 nice, I see this removes the floats on the y-axis when there are only two meetings, for example. So the y-axis just has 1 & 2, rather than 1.10, 1.20, 1.30...

Reply:
Yes, you can't have 1 and a half meeting :)